content/events/2020-amsterdam/_index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Amsterdam
+date: '2020-09-24'
+end: '2020-09-25'
+location:
+  country: The Netherlands
+  city: Amsterdam
+  url: https://www.amsterdam.nl/
+  venue: Westergasterras
+map: amsterdam-map.png
+sched_slug: kubernetescommunitydayams
+social:
+  facebook: https://facebook.com/cloudnativeams
+  twitter: 'https://twitter.com/cloudnativeams'
+sponsors:
+- level: Initial committers
```

Comment:
Please change to something like `Launch Sponsors` or `Initial Sponsors`, not Committers, which has a coding connotation.

Reply:
addressed in 098fe22